plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/GoogleLoginCommandHandler.java

```diff
@@ -21,19 +21,14 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
 
     Credential credential = loginService.getCachedActiveCredential();
```

Comment:
This code could use some comments explaining how it works. (I.e. pressing the button logs you out if you're already logged in; otherwise it logs you in.) I see it now, but I can imagine it's going to confuse me in 6 months. 

I do wonder if we need a different UI, e.g. swapping out the button; but that can wait until we design multilogin.